src/main/java/com/sproutsocial/nsq/Subscriber.java

```diff
@@ -104,15 +104,18 @@ public synchronized void subscribe(String topic, String channel, int maxInFlight
      * call.
      */
     public synchronized boolean unsubscribe(String topic, String channel) {
+        return unsubscribeSubscription(topic, channel) != null;
+    }
+
+    synchronized Subscription unsubscribeSubscription(String topic, String channel) {
```

Comment:
purely split out for testing purposes?

Reply:
Yes, and it's package private. Don't want to leak the details of the internal `Subscription` object.